e2e_testing/torchscript/basic.py

```diff
@@ -493,3 +534,99 @@ def forward(self, x):
 @register_test_case(module_factory=lambda: ContiguousModule())
 def ContiguousModule_basic(module, tu: TestUtils):
     module.forward(tu.rand(3, 1))
+
+# ==============================================================================
+
+class SliceModule(torch.nn.Module):
+    def __init__(self):
+        super().__init__()
+
+    @export
+    @annotate_args([
+        None,
+        ([-1, -1, -1], torch.float32, True),
+    ])
+    def forward(self, x):
+        return x[0:5:1, :, 0:4:1]
```

Comment:
just a note that we will need more test cases like different `start` and `step` values. also edge case like `start` == `end`.